keepassxc-browser/background/browserAction.js

```diff
@@ -202,6 +202,14 @@ browserAction.removeRememberPopup = function(callback, tab, removeImmediately) {
 browserAction.setRememberPopup = function(tabId, username, password, url, usernameExists, credentialsList) {
     browser.storage.local.get({'settings': {}}).then(function(item) {
         const settings = item.settings;
+
+        // Don't show anything if the site is in the ignore list
+        for (const site in settings.ignoredSites) {
```

Comment:
Can't we read this into a O(1) map or something?

Reply:
AFAIK this is the quickest way to sort that kind of Object list.

Reply:
I meant to read it into a map once and then reference from that. But perhaps not that important anyway.